sdk/resources/Microsoft.Azure.Management.Resource/src/Generated/SubscriptionClient.cs

```diff
@@ -323,6 +330,7 @@ public SubscriptionClient(System.Uri baseUri, ServiceClientCredentials credentia
         /// </summary>
         private void Initialize()
         {
+            Operations = new Operations(this);
```

Comment:
This looks to be the problem - the Operations classes are specific to a client.  There isn't any real reason why the operations for listing operations need to be here on the subscription client - you should simply be able to remove this, either in swagger, swagger directive, or by hand.

Reply:
PR for fix is here: https://github.com/Azure/azure-rest-api-specs/pull/14853